.travis.yml

```diff
@@ -72,23 +72,25 @@ jobs:
         # setup sitecustomize so we can make all subprocess start coverage
         - export PYSITEDIR=$(python -c "import sys; print(sys.path[-1])")
         - export COVERAGE_PROCESS_START="$PWD/.coveragerc"
+        - export COVERAGE_FILE="$PWD/.coverage"
         - mkdir -p "$PYSITEDIR"
```

Comment:
any chance I can convince you to just make this another shell script in .travis
And/or update the .coveragrc which is already in .travis?

Reply:
yeah I can move this into shell scripts. The main reason for not just having it all static in a .coveragerc was so the absolute path from $PWD would make it into the .coveragerc file

Reply:
maybe a shell script could generate the .coveragerc with some replace logic for the PWD? Would be cleaner then having everything in a script

Reply:
I think the powershell steps can be scripted as well.

Example from another repo where they do:

```
build_script:
  - ps: .\build.ps1 --bootstrap
  - ps: .\build.ps1 -Target ContinuousIntegration
```

https://github.com/chocolatey/ChocolateyGUI/blob/develop/.appveyor.yml
https://github.com/chocolatey/ChocolateyGUI/blob/develop/build.ps1

Seems like having many lines of shell logic in the .travis and .appveyor is hard to read/maintain/debug..